lib/events.js

```diff
@@ -35,5 +35,12 @@ export const moneyCanMoveFromEvent = event => {
 
 export const sortEvents = events => {
   const getDate = column => event => !event[column] ? null : new Date(event[column]);
-  return orderBy(events, [getDate('startsAt'), getDate('endsAt'), 'id'], ['desc', 'desc', 'desc']);
+  const eventPartitions = partition(events, isPastEvent);
+  const pastEvents = eventPartitions[0];
+  const presentEvents = eventPartitions[1];
```

Comment:
:ice_cream: Array destructuring FTW

```suggestion
  const [pastEvents, presentEvents] = partition(events, isPastEvent);
```

Reply:
Thanks. corrected it. 👍🏽